cpp/cmake_modules/DefineOptions.cmake

```diff
@@ -108,7 +108,10 @@ if("${CMAKE_SOURCE_DIR}" STREQUAL "${CMAKE_CURRENT_SOURCE_DIR}")
 
   # Arm64 architectures and extensions can lead to exploding combinations.
   # So set it directly through cmake command line.
-  define_option_string(ARROW_ARMV8_ARCH "Arm64 arch and extensions" "armv8-a+crc+crypto")
+  define_option_string(ARROW_ARMV8_ARCH
+                       "Arm64 arch and extensions"
+                       "" # default to None
```

Comment:
We can change the default value to "armv8-a"

Reply:
I'll check this out on my rockpro64 and then make the changes below if all looks good